setup.py

```diff
@@ -30,6 +30,7 @@
     },
 
     install_requires=[
-        'pylint>=1.4.0',
+        'pylint==1.4.4',
+        'pylint-django==0.6.1',
```

Comment:
requirements.txt is for developers working on edx-lint.  Setup.py is for people using edx-lint.  I thought this was a well-established pattern?  I guess not.